cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapperWrapper.java

```diff
@@ -159,7 +159,7 @@ public boolean nextKeyValue() throws IOException, InterruptedException {
         boolean result = super.nextKeyValue();
         if (++processedRecords > flushFreq) {
           try {
-            LOG.info("Flushing dataset operations...");
+            LOG.trace("Flushing dataset operations...");
```

Comment:
@chtyim Isn't it unnecessary to log it, then? It will be the same value each time, and it'll just be whatever user has configured.